core-services/release-controller/_releases/priv/release-ocp-4.6.json

```diff
@@ -51,6 +51,13 @@
                 "name": "release-openshift-ocp-installer-e2e-aws-upi-4.6-priv"
             }
         },
+        "aws-workers-rhel7": {
+            "disabled": true,
```

Comment:
You said you wanted to be blocking, which is not what this is doing

Reply:
These were all disabled, so I left it as is.

Reply:
We've decided that workers-rhel7 should be informing.